Properties/Resources.resx

```diff
@@ -0,0 +1,247 @@
+  </data>
+  <data name="UnableToDetermineFileType_file" xml:space="preserve">
+    <value>Failed to determine type of the '{0}' file.</value>
+    <comment>{0} - A filename</comment>
+  </data>
+  <data name="FailedToReadSymLink_link_error" xml:space="preserve">
+    <value>Could not read the symbolic link '{0}': {1}.</value>
+    <comment>{0} - The symbolic link that was being read.
+{1} - The actual error message reported by the operating system.</comment>
+  </data>
+  <data name="FileTypeNotSupported_filetype" xml:space="preserve">
+    <value>Storing files of type {0} is not supported.</value>
+    <comment>{0} - The type of the file.</comment>
+  </data>
+  <data name="FailedToSetOwner_entry_error_oserror" xml:space="preserve">
+    <value>Warning: failed to set owner of entry '{0}' ({1}:{2}).</value>
+    <comment>{0} - A filename.
+{1} - The error message.
+{2} - The operating system error.</comment>
+  </data>
+  <data name="FailedToStatFile_file_error" xml:space="preserve">
+    <value>Warning: failed to stat file '{0}': {1}.</value>
+    <comment>{0} - A filename.
+{1} - The error message.</comment>
+  </data>
+  <data name="UnexpectedOptionsType_string_type_type" xml:space="preserve">
+    <value>Unexpected options type for {0}. Expected {1} but got {2}.</value>
+    <comment>{0} - The parameter name.
+{1} - The expected type of the parameter.
+{2} - The actual type of the parameter.</comment>
+  </data>
+  <data name="FileAlreadyExists" xml:space="preserve">
+    <value>The file already exists.</value>
+    <comment></comment>
+  </data>
+  <data name="FilePathAlreadyExists_file" xml:space="preserve">
+    <value>File {0} already exists.</value>
+    <comment>{0} - A filename.</comment>
+  </data>
+  <data name="OutOfMemory" xml:space="preserve">
+    <value>libzip returned out of memory error.</value>
+    <comment></comment>
+  </data>
+  <data name="StreamFailedConsistencyChecks" xml:space="preserve">
+    <value>The stream failed consistency checks.</value>
+    <comment></comment>
+  </data>
+  <data name="StreamDoesNotExist" xml:space="preserve">
+    <value>Stream does not exist and file creation wasn't requested.</value>
+    <comment></comment>
+  </data>
+  <data name="StreamIsNotAZip" xml:space="preserve">
+    <value>Stream is not a ZIP archive.</value>
+    <comment></comment>
+  </data>
+  <data name="StreamCouldNotBeOpened" xml:space="preserve">
+    <value>Stream could not be opened.</value>
+    <comment></comment>
+  </data>
+  <data name="ErrorReadingStream" xml:space="preserve">
+    <value>Error occured while reading the Stream.</value>
+    <comment></comment>
+  </data>
+  <data name="StreamDoesNotSupportSeeking" xml:space="preserve">
+    <value>Stream does not support seeking.</value>
+    <comment></comment>
+  </data>
+  <data name="UnexpectedLibZipError_error" xml:space="preserve">
+    <value>Unexpected libzip error: {0}.</value>
+    <comment>{0} - An error message from libzip.</comment>
```

Comment:
Maybe to add a comment for safety since `libzip` is lowercase:

```
<comment>The following is the name of a library and should not be translated: libzip
{0} - An error message from libzip.</comment>
```